metadata-models/src/main/pegasus/com/linkedin/common/fieldtransformer/IdentityTransformer.pdl

```diff
@@ -0,0 +1,8 @@
+namespace com.linkedin.common.fieldtransformer
+
+/**
+ * Field transformation expressed as Identity function
+ */
+record IdentityTransformer {
```

Comment:
not sure about this empty model. Are you planning to add fields here in the future?

Reply:
We could potentially add more metadata if needed in future.

Reply:
I can't think of any metadata that you would add here. I am leaning towards defining one model which has two fields - `type` and `fqcn`. `type` will be an enum  'BlackBox, Identity and UDF` and for `fqcn` we can define udf specific class name. Having the empty models doesn't look like it is adding any extra benefit and it doesn't look clean. WDYT?

Reply:
Discussed offline.

This is another modelling challenge :slightly_smiling_face: Here is the thing. Type here is an enum.

1. Few fields of the enum are sufficient on their own (IDENTITY, BLACKBOX)
2. Few fields of the enum need extra metadata along with it (UDF)

So, we are debating on what is the best way to represent this scenario.

UDF enum at some point in future can get extended with extra metadata.
So putting fqcn (2) in the same models as (1) can soon be confusing for producers and consumers. While segregating them into different models adds flexibility, but we end up with few  empty models for now, hoping to add some metadata in far future.


@hshahoss  and me are inclined to go with the approach in this PR.

Reply:
How about a union between enum & record (for UDF)?

Reply:
I believe you are referring to
 
```pdl
transformation : union[FieldLineageTypeEnum, UDFTransformation] 
```

I like this, we can get away with empty models and still achieve strongly typed models.